rotkehlchen/tests/unit/uniswap/conftest.py

```diff
@@ -0,0 +1,39 @@
+# pylint: disable=redefined-outer-name
```

Comment:
Why are you disabling those? Where are you getting them?

Reply:
The reason is that my fixture name is `uniswap` but I have the following import:
```python
from rotkehlchen.chain.ethereum.uniswap.uniswap import Uniswap
```
And I get:
```shell
$ pylint rotkehlchen/tests/unit/uniswap/conftest.py
************* Module rotkehlchen.tests.unit.uniswap.conftest
rotkehlchen/tests/unit/uniswap/conftest.py:1:0: C0114: Missing module docstring (missing-module-docstring)
rotkehlchen/tests/unit/uniswap/conftest.py:10:0: C0116: Missing function or method docstring (missing-function-docstring)
rotkehlchen/tests/unit/uniswap/conftest.py:22:4: W0621: Redefining name 'uniswap' from outer scope (line 10) (redefined-outer-name)
rotkehlchen/tests/unit/uniswap/conftest.py:32:0: C0116: Missing function or method docstring (missing-function-docstring)

-------------------------------------------------------------------
Your code has been rated at 6.92/10 (previous run: 10.00/10, -3.08)
```
I can rename the fixture name, though.

Reply:
yes in this case you should rename the fixture.

Also don't run pylint on its own like that. Run it with the arguments we have at make lint. There is a lot of stuff that pylint warns for and we don't care and have disabled. https://github.com/rotki/rotki/blob/develop/Makefile#L4

Reply:
Ok, done. Removed the unnecessary disabling comments and renamed the fixture.

Reply:
thanks